radboudumc-template/templates/new/publications-author.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% block title %}{{ page.title }}{%endblock%}
+{% block content %}
+    {% include 'blocks/breadcrumbs.html' %}
+    <div class="container">
+    <h1>{{ page.title }}</h1>
+
+    <p class="my-4">
+        <a href="{{SITEURL}}/members/{{ page.author}}/">Back to page of {{ page.author_name}}</a>
+    </p>
+
+    {% set list_years = [] %}
+    {% set list_pub_type = [] %}
+
+    {%if bibkeys_html %}
+    {% for bibkey in bibkeys_html.keys() %}
+        {# Getting list of years #}
+        {% if bibkeys_html[bibkey]['year'] not in list_years %}
+            {% if list_years.append(bibkeys_html[bibkey]['year']) %}{% endif %}
+        {% endif %}
+        {# Getting list of pub_type #}
+        {% if bibkeys_html[bibkey]['pub_type'] not in list_pub_type %}
+            {% if list_pub_type.append(bibkeys_html[bibkey]['pub_type']) %}{% endif %}
+        {% endif %}
+    {% endfor %}
+    {% endif %}
+
+    {% for cur_year in list_years|sort(reverse = True) %}
+            {% set found_year = { 'flag': false } %}
+            {% for pub_type in list_pub_type %}
+                {% set found = { 'flag': false } %}
+                {% for bibkey in bibkeys_html.keys() %}
+                    {# TODO: Lazy code alert! I copied publications.html template and added page.bibkeys.split(',') to make this work for publications per researcher #}
```

Comment:
Can't we make a template of this? I.e. only have 1 block for generating the publication lists.

Reply:
made a block for this